src/GitInfo/build/GitInfo.targets

```diff
@@ -579,7 +587,7 @@
 			DependsOnTargets="_GitBranch;_GitCommit"
 			Condition="'$(GitBaseVersion)' == '' And '$(GitIgnoreTagVersion)' != 'true' ">
 
-		<Exec Command='$(GitExe) describe --tags --abbrev=0'
+		<Exec Command='$(GitExe) describe --tags --match=$(GitTagRegex) --abbrev=0'

```

Comment:
does this `--match` argument support our current semver2 expression? If so, then we just need to pass it and that's it? If it breaks, do we need a simpler format of the semver2 regex?

Reply:
The answer is, I really don't know for sure (I don't really have a good way to test on my side).  I suspect it does.  This question was the impetus behind adding a second regex rather than just using the _GitBaseVersionExpr.